src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
-body {
-  margin: 0;
+@import './variables';
+@import './reset';
+@import './header';
+@import './card';
+@import './stars';
+
+.container {
+  padding-top: 50px;
+  padding-left: 40px;
+  padding-right: 40px;
+  gap: 46px 48px;
+  grid-template-columns: repeat(1, 1fr);
+  max-width: 1024px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)